config/initializers/sentry.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.config.to_prepare do
   # Sentry is enabled if SENTRY_DSN environment variable is set
   Sentry.init do |config|
+    config.rails.report_rescued_exceptions = true
```

Comment:
I think the default is true anyways?
https://docs.sentry.io/platforms/ruby/guides/rails/configuration/integration_options/#sentry-rails

Reply:
Yes it is the default, I thought I'd put in anyway to save me looking in the docs to confirm next time :)